examples/tv-casting-app/tv-casting-common/tv-casting-app.matter

```diff
@@ -2459,11 +2459,66 @@ server cluster WindowCovering = 258 {
 
 
 endpoint 0 {
+  instantiate cluster AdministratorCommissioning;
+  instantiate cluster Basic;
+  instantiate cluster Binding;
+  instantiate cluster Descriptor;
+  instantiate cluster DiagnosticLogs;
+  instantiate cluster EthernetNetworkDiagnostics;
+  instantiate cluster FixedLabel;
+  instantiate cluster GeneralCommissioning;
+  instantiate cluster GeneralDiagnostics;
+  instantiate cluster GroupKeyManagement;
+  instantiate cluster LocalizationConfiguration;
+  instantiate cluster NetworkCommissioning;
+  instantiate cluster OtaSoftwareUpdateProvider;
+  instantiate cluster OperationalCredentials;
+  instantiate cluster RelativeHumidityMeasurement;
+  instantiate cluster SoftwareDiagnostics;
+  instantiate cluster ThreadNetworkDiagnostics;
+  instantiate cluster UserLabel;
+  instantiate cluster WiFiNetworkDiagnostics;
 }
 
 endpoint 1 {
```

Comment:
@chrisdecenzo - are these correct? Why does a TV casting app have things like DoorLock or PressureMeasurement on endpoints?

Reply:
Note that this has nothing to do with this PR ... but this PR exposes what is in a ZAP file in a more readable way. Or at least it tries to - if it is  wrong please let me know.